### PR TITLE
Convert bluetooth tests to ztest

### PR DESCRIPTION
**Pull request by punitvara**

### CODE REVIEW

**Comment by punitvara** [comment]
@inakypg Why am I not getting PR message any idea ?

File: tests/bluetooth/bluetooth/src/bluetooth.c [hunk 1]
```diff
@@ -41,22 +42,18 @@ static void driver_init(void)
...
-	ret = bt_enable(NULL);
-	if (ret == EXPECTED_ERROR) {
-		ret_code = TC_PASS;
-	} else {
-		ret_code = TC_FAIL;
-	}
-
-	TC_END(ret_code, "%s - %s.\n", ret_code == TC_PASS ? PASS : FAIL,
-		   __func__);
+	zassert_true(bt_enable(NULL),
```
Comment (jhedberg) [inline tests/bluetooth/bluetooth/src/bluetooth.c]: shouldn't this be zassert_true(bt_enable(NULL) == EXPECTED_ERROR

**Review by jhedberg — CHANGES_REQUESTED** [review]

File: tests/bluetooth/bluetooth/src/bluetooth.c (hunk 1 above)
Comment (punitvara) [reply tests/bluetooth/bluetooth/src/bluetooth.c]: Opps... Thanks catching it.

**Review by Vudentz — DISMISSED** [review]
LGTM

**Review by nashif — APPROVED** [review]

**Pull request merged**